src/session/al-session.ts

```diff
@@ -57,6 +57,11 @@ export interface AlSessionOptions {
      * If account metadata is resolved, should the client use the consolidated/gestalt resolver endpoint?  Defaults to `false`.
      */
     useConsolidatedResolver?:boolean;
+
+    /**
+     * Should FOX (features, options, and experiences) profile be retrieved as part of account metadata?  Defaults to `false`.
+     */
+    useFox?:boolean;
```

Comment:
FOX is an experimental endpoint -- it will work in integration but not in master -- so we need to make sure that the useFox setting is defaulted to false and respected.  This can be as simple as returning an empty `AlFoxSnapshot` if `useFox` is false [here](https://github.com/alertlogic/nepal-core/pull/41/files#diff-12f699a268fc35883d97c40c1b4628e9R723).

Reply:
You got it
https://github.com/alertlogic/nepal-core/pull/41/commits/5fc6f0d8d5846a342ca589cd47f273126eebf665#diff-12f699a268fc35883d97c40c1b4628e9R724